src/Raven.Server/NotificationCenter/Handlers/DatabaseNotificationCenterHandler.cs

```diff
@@ -6,11 +6,42 @@
 using Raven.Server.Documents;
 using Raven.Server.NotificationCenter.Notifications;
 using Raven.Server.Routing;
+using Raven.Server.ServerWide.Context;
+using Raven.Server.Utils;
+using Sparrow.Json;
 
 namespace Raven.Server.NotificationCenter.Handlers
 {
     public class DatabaseNotificationCenterHandler : DatabaseRequestHandler
     {
+        [RavenAction("/databases/*/notification-center/get", "GET", AuthorizationStatus.ValidUser, EndpointType.Read, SkipUsagesCount = true)]
+        public async Task GetNotifications()
+        {
```

Comment:
Please provide parameter to determine type of notifications to get (alerts, performance hints)

Reply:
done